CMakeLists.txt

```diff
@@ -0,0 +1,136 @@
+cmake_minimum_required(VERSION 3.20)
+project(vkfw LANGUAGES CXX C)
+
+# Build options.
+option(VKFW_DEV_BUILD "Build VKFW for development" OFF)
+option(VKFW_INSTALL "Create install target" OFF)
+option(VKFW_BUILD_EXAMPLES "Build examples" OFF)
+
+# Configuration options
```

Comment:
It's probably worth adding something like "see `README.md` for details" to this comment.